roles/openshift_node/templates/node.yaml.v1.j2

```diff
@@ -15,7 +15,7 @@ kind: NodeConfig
 kubeletArguments: {{ openshift.node.kubelet_args | to_json }}
 {% endif %}
 masterKubeConfig: system:node:{{ openshift.common.hostname }}.kubeconfig
-{% if openshift.common.use_openshift_sdn %}
+{% if openshift.common.use_openshift_sdn or openshift.common.use_nuage %}
 networkPluginName: {{ openshift.common.sdn_network_plugin_name }}
```

Comment:
Not required for the node config files. But perhaps still needed for master-config.yaml.